yolov8_seg_ros:
  image_topic: "/usb_cam/image_raw"        # image topic
  target_size: 320
  prob_threshold: 0.4
  nms_threshold: 0.5
  use_gpu: false
  downsampling_infer: 5                   # inference every n frames, if 3, then 1/3 of images
